homeassistant/components/calendar/__init__.py

```diff
@@ -69,6 +74,18 @@ def __init__(self, hass, data):
             'description': '',
         }
 
+        class CalendarEventView(http.HomeAssistantView):
+            """View to retrieve calendar content."""
+
+            url = '/api/calendar/' + self.entity_id.split(".")[-1]
```

Comment:
I find it weird that the API is bound to entities.

Reply:
I don't have any other idea :/ do you have one ?